package.json

```diff
@@ -13,7 +13,8 @@
   },
   "devDependencies": {
     "@wordpress/env": "^9.10.0",
-    "@wordpress/scripts": "^14.0.0"
+    "@wordpress/scripts": "^14.0.0",
+    "prettier": "3.3.3"
```

Comment:
Given we have a mixed project that includes files in languages like JavaScript, HTML, or CSS, alongside our PHP files, it seems fine to consider using both tools (prettier and phpcs). In our case, Prettier could be used for formatting your non-PHP files, while phpcs handles your PHP code. This approach allows us to leverage the strengths of each tool, ensuring that all parts of your project adhere to the highest coding standards.
Lets get a feedback from vendor as well on this as we should be very careful while adding additional dependency on 3P package.